src/dev/src/app/components/tree-view/tree-view-client-side-demo.ts

```diff
@@ -347,13 +368,18 @@ export class TreeViewClientSideDemo {
 
   treeNodeData: TreeNodeData<FoodNode> = {
     data: treeData,
-    nodeLabel: treeData.name,
+    nodeLabel: treeData.name
   };
   serverSideTreeNodeData: TreeNodeData<FoodNode> = {
     data: serverData,
-    nodeLabel: serverData.name,
+    nodeLabel: serverData.name
   };
   noRoot = NonRootTreeNode.instance;
+  noRootAutoSelector: TreeViewAutoSelector<FoodNode> = {
```

Comment:
Why do we need two kind of autoSelector ? They both do the same ! We should use the same autoSelector internally

Reply:
Fixed